src/routes/organization/[organizationId]/+page.svelte

```diff
@@ -96,24 +97,41 @@
   };
 
   const cancelInvite = (inviteId: string) =>
+    confirm('Are you sure you want to cancel this invite?') &&
```

Comment:
Many small styling and quality-of-life fixes on this page. It's like 10% better now, still a ways to go

Reply:
This is how it looks now
![image](https://github.com/user-attachments/assets/dacdca6e-ee06-4901-b804-809cfb01df0e)